KNOWN_ISSUES.md

```diff
@@ -0,0 +1,10 @@
+# Known Issues
+
+## Can i run this on AMD/METAL GPUs?
+> TODO
```

Comment:
"We'll support many other GPUs as soon as the CUDA version is sufficiently stable"